.github/workflows/clone-docker-image-action.yaml

```diff
@@ -43,25 +43,6 @@
           required: true
   
   jobs:
-    check-image-exists:
```

Comment:
As implemented, this check doesn't really provide any value because it's just checking to see if `latest` exists, and not that the tag that needs to be copied actually exists.  I decided to remove it entirely, since the next step which actually tries to copy the image will fail if the image does not exist.